packages/vite/package.json

```diff
@@ -19,7 +19,10 @@
   "scripts": {
     "build": "yarn build:js && yarn build:types",
     "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\"",
-    "build:types": "tsc --build --verbose"
+    "build:types": "tsc --build --verbose",
+    "test": "yarn test:node && echo",
+    "test:node": "glob './src/**/__tests__/*.test.mts' --cmd='node --loader tsx --no-warnings --test'",
+    "test:watch": "glob './src/**/__tests__/*.test.mts' --cmd='node --loader tsx --no-warnings --test --watch'"
```

Comment:
Can't just do `"test:watch": "yarn test --watch"` because the `&& echo` blocks it
Also can't do `"test:watch: "yarn test:node --watch"` because `--watch` ends up as an argument to `glob`, not `node`.
So typing it all out was the only way I could make it work

Reply:
@Tobbe this is great, wanted to throw out `NODE_OPTIONS` here cause it may let you pass watch: https://nodejs.org/api/cli.html#node_optionsoptions

I think it'd be like `"test:watch": "NODE_OPTIONS='--watch' yarn test:node"`

Reply:
With these scripts:
```
  "scripts": {
    "build": "yarn build:js && yarn build:types",
    "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\"",
    "build:types": "tsc --build --verbose",
    "test": "yarn test:node && echo",
    "test:node": "glob './src/**/__tests__/*.test.mts' --cmd='node --loader tsx --no-warnings --test'",
    "test:watch": "NODE_OPTIONS='--watch' yarn test:node"
  },
```
I get this error when I try `yarn test:watch`
```
/usr/local/bin/node: --watch requires specifying a file
```